Pyjion/absint.cpp

```diff
@@ -71,15 +72,39 @@ bool AbstractInterpreter::preprocess() {
     int oparg;
     vector<bool> ehKind;
     vector<AbsIntBlockInfo> blockStarts;
-    for (size_t curByte = 0; curByte < m_size; curByte++) {
-        auto opcodeIndex = curByte;
-        auto byte = m_byteCode[curByte];
-        if (HAS_ARG(byte)) {
-            oparg = NEXTARG();
-        }
+    for (size_t curByte = 0; curByte < m_size; curByte += sizeof(_Py_CODEUNIT)) {
```

Comment:
It needs to be size_t because it's the index...  If it was a _Py_CODEUNIT* that pointed to our byte array we could do that, but in general the byte code treats these as offsets into the byte array, not offsets into the _Py_CODEUNIT array.  For example JUMP_ABSOLUTE provides an absolute position in the array, JUMP_RELATIVE is the number of bytes to jump forward, etc...  So for better or worse it makes the most sense to just deal with byte offsets everywhere